src/mcts/search.cc

```diff
@@ -36,8 +36,10 @@ namespace lczero {
 const char* Search::kMiniBatchSizeStr = "Minibatch size for NN inference";
 const char* Search::kMaxPrefetchBatchStr = "Max prefetch nodes, per NN call";
 const char* Search::kCpuctStr = "Cpuct MCTS option";
-const char* Search::kTemperatureStr = "Initial temperature";
-const char* Search::kTempDecayMovesStr = "Moves with temperature decay";
+const char* Search::kMoveTempStr = "Initial move-decay temperature";
```

Comment:
That's fine to have that zoo of params for testing, but after few days we should reduce it.
Was it tested that having eval-decay actually helps with anything, or it's purely theoretical speculation?

Reply:
Well we need *some* eval-decay if/when we ever try this on main net. It's like resign threshold, the correct value may change. (And at any rate, "actual" testing hasn't happened yet. This would need to be merged first, followed by a run on the test pipeline before we can make any conclusions about reasonable eval-decays are.)